jobs/service-fabrik-broker/templates/bin/service-fabrik-broker_ctl.erb

```diff
@@ -22,6 +22,13 @@ case $1 in
     fi
     /usr/sbin/service rsyslog restart
 
+    #Load deployment actions
+    echo "starting to load deployment action scripts"
+    ${NODE_CMD} ${PACKAGE_DIR}/lib/LoadDeploymentActions.js \
+      1>> ${LOG_DIR}/${OUTPUT_LABEL}.stdout.log \
+      2>> ${LOG_DIR}/${OUTPUT_LABEL}.stderr.log
+    echo "scripts loaded successfully"
```

Comment:
might want to change this echo to match the echo on line 26

Reply:
Done